why_postgres.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Why a new Postgres platform?'
```

Comment:
Also CC-ing @snide because he knows HN best.

Reply:
I like the 2nd one but would tweak slightly to "The world doesn't need another Postgres service, they need a Postgres database platform"

Other click-baity suggestions:

- We Exposed PostgreSQL (And Here's Why You Should Too)
- Is Postgres Enough? Xata Reimagines the Future 
- Introducing a Postgres Service Your Devs Will Actually Love
- Evolving Postgres Beyond the Database
- Stop Managing Postgres, Focus on Your App

Reply:
Missed this one from @CezzaineZ because I read the bullet points. "The world doesn't need another Postgres service, they need a Postgres database platform"

Possible slant for brevity.

"You don't need a Postgres host, you need a Postgres Platform"

Reply:
Adding in one other idea that I got after reading some of these:

"Postgres can't do everything"
"Postgres can't do everything, but a platform for Postgres can"

Reply:
So I have another draft blog post that goes into that direction of "Postgres can't do everything" and pros/cons for "Postgres for everything" versus "Right tool for the job". But I feel like this one should be a bit more like an announcement, we're launching a new Postgres service, with the twist of the data platform.

Reply:
How about something along the lines of:
-  "A new Postgres platform" 
-  "A new serverless Postgres platform"
- "Xata, a new Postgres platform"
- "Xata, a new serverless Postgres platform"

It's not controversial but has novelty, which also tends to do well. An we could do a "Show HN" post for it, I think that might be ok as per the HN rules.

Reply:
My suggestion: 

For HN:
"How we built the free tier in our Postgres service to be cost efficient, with HA and no pausing"
For blog post: 
"How we built our new Postgres service"

Reply:
I wouldn't bet on the "database platform" as people don't know what it means. I think just calling it a Postgres service is better understood.